lambdas/archive-vcs/src/main/resources/IpvLambdaJsonLayout.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Is this identical for the other lambda replaying cimit ci's?

Reply:
@MikeCollingwood I believe these are identical/very similar for all of our lambdas and are put alongside the lambda code for the purposes of bundling the lambda artifacts for AWS